giraffe/src/components/SizedTable.tsx

```diff
@@ -91,6 +93,16 @@ export const SizedTable: FunctionComponent<Props> = ({
                   />
                 )
 
+              case LayerTypes.SimpleTable: {
+                return (
+                  <SimpleTableLayer
+                    config={
```

Comment:
The `key` prop is missing here. If you don't pass `key` React will give you a nice bright red warning in the console when `SimpleTable` gets rendered by SizedTable.